### PR TITLE
Merge develop to main (shared theme)

### PR DESCRIPTION
**Pull request by Polynya**
## Call out any relevant implementation decisions
- Are there any links to back up your chosen methodology?
- Why have you taken the approach?
- Any particular problem areas?
## This PR has been tested in the following browsers
- [ ] Arc
- [ ] Edge
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] IE 11 (Windows)
- [ ] iOS Chrome
- [ ] iOS Safari
- [ ] Android Chrome
- [ ] Android Firefox
- [ ] Android default

### CODE REVIEW

File: web/modules/custom/ecc_d10/README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# Temporary modules for Drupal 10 upgrade&#10&#10Delete this folder after the Drupal 10 upgrade.
\ No newline at end of file
+# Temporary modules for Drupal 10 upgrade
```
Comment (Polynya) [inline web/modules/custom/ecc_d10/README.md:1]: These modules will be deleted as part of LP-59 (Tidy up essex-gov-uk-drupal repository)

File: config/default/system.site.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ slogan: ''
 page:
   403: /node/1212
   404: /node/1212
-  front: /node/1211
+  front: /node/113
```
Comment (siliconmeadow) [inline config/default/system.site.yml:11]: Config ignore? I just checked prod and it's still `1211`

**Review by siliconmeadow — COMMENTED** [review]
Just the one thing about the front page, @Polynya.

File: config/default/system.site.yml (hunk 2 above)
Comment (Polynya) [reply config/default/system.site.yml:11]: I'll revert this. The next release with have the new version of Config Ignore, and we can stop this from being exported.

**Pull request merged**